tests/foreman/ui/test_discoveryrule.py

```diff
@@ -61,8 +66,8 @@ def setUpClass(cls):
         cls.saved_per_page = str(cls.per_page.value)
         cls.per_page.value = '100000'
         cls.per_page.update({'value'})
-
-        cls.host_group = entities.HostGroup().create()
+        cls.host_group = entities.HostGroup(
+            organization=[cls.session_org]).create()
```

Comment:
it doesn't seem as obligatory thing as functionality works just fine without it. Should I add it anyway?